docs/products/redis/howto/estimate-max-number-of-connections.rst

```diff
@@ -0,0 +1,23 @@
+Estimate maximum number of connection in Aiven for Redis
+========================================================
+
+The number of simultaneous connections in Aiven for Redis depends on the total available memory on the server. 
+
+
+Here is how you can estimate it:
```

Comment:
Maybe move away from "Here" -> You can use the following to estimate it:

Reply:
Yes, I think is better too. Applied! https://github.com/aiven/devportal/pull/423/commits/18b987e7dde55430c54200ce439806f5191d3bbe